workflows/toast_so_sim.py

```diff
@@ -402,6 +415,18 @@ def simulate_data(job, args, toast_comm, telescope, schedule):
         timer=timer,
     )
 
+    # Optionally write out the data
+    if args.out_dir is not None:
+        hdf5_path = os.path.join(args.out_dir, "data")
+        if toast_comm.world_rank == 0:
```

Comment:
Thanks, this is a good idea to put it in a subdirectory and create that on one process.